src/cplus_plugin/models/constant_raster.py

```diff
@@ -0,0 +1,220 @@
+# -*- coding: utf-8 -*-
+"""Models for Constant Raster."""
+
+from __future__ import annotations
+import dataclasses
+import typing
+
+from .base import NcsPathway, PriorityLayerType
+from ..definitions.constants import (
+    MIN_VALUE_ATTRIBUTE,  # "minimum_value"
+    MAX_VALUE_ATTRIBUTE,  # "maximum_value"
+    REMOVE_EXISTING_ATTRIBUTE,  # "remove_existing"
+    NCS_PATHWAY_IDENTIFIER_PROPERTY,  # "pathway_identifier"
+    ENABLED_ATTRIBUTE,  # "enabled"
+    PATH_ATTRIBUTE,  # "path"
+)
+
+
+# ---------------------------
+# String-backed enums (stable)
+# ---------------------------
+class _StrEnum(str):
+    def __str__(self) -> str:  # pragma: no cover
+        return str(self.value)
+
+
+class ScaleMode(_StrEnum):
+    AUTO_MINMAX = "auto_minmax"
+    MANUAL_MINMAX = "manual_minmax"
+
+
+# ---------------------------
+# Data models
+# ---------------------------
+@dataclasses.dataclass
+class ConstantItem:
+    """One constant definition for a single pathway."""
+
+    pathway: NcsPathway
```

Comment:
Make it polymorphic by having this attribute as `LayerModelComponentType` since it can be used to refer to either an NCS pathway or Activity.